pkg/utils/utils.go

```diff
@@ -413,3 +414,57 @@ func GetAKONamespace() string {
 	}
 	return AKO_DEFAULT_NS
 }
+
+func GetTokenFromRestObj(robj interface{}, ctrlAuthToken string) (oldTokenID string, refresh bool, err error) {
+	oldTokenID = ""
+	refresh = false
+	err = nil
+	parseError := errors.New("Failed to parse token response obj")
+
+	if _, ok := robj.(map[string]interface{}); !ok {
+		err = parseError
```

Comment:
It would be also good to dump the raw response IMHO, if there's a parse error. This will give an idea to the users while debugging to figure out what went wrong.

Reply:
added the response to error log